content/software-triage.md

```diff
@@ -46,44 +46,8 @@ Do the symptoms go away if the system is suspended, rebooted or shut down and th
     E.g. windows frozen in place, windows will not resize. Stuck in full-screen mode, etc.
     Mouse freezes intermittently or stutters it moves across the screen, or keyboard input lags.
 
-## Causes
-
-- GNOME Extension bugs.
-- Bugs inherent in applications.
-- Corrupted software updates/installs.
-    Files may have been corrupted as they were downloaded, for example, by a network interruption.
-- Out of date OS libraries.
-    Many applications refer to system libraries. Some programs may be expecting to find a specific version of a library.
-- Out of date drivers.
-    Older kernels may be using out of date driver software.
-- Incorrect Drivers.
-    If an unneeded or incorrect driver is installed this can cause unexpected side effects.
-- Kernel Bugs
-
 ## Solutions
 
-We troubleshoot issues with the approach of taking small steps, then gradually larger steps:
```

Comment:
I think its important for customers to see the logic in how we're troubleshooting so that when we follow these same procedures on the ticket they're already primed for our way of thinking.